msoms/1.9/README.md

```diff
@@ -18,7 +18,7 @@ Analytics](https://azure.microsoft.com/en-us/documentation/articles/log-analytic
 Setting up OMS from the DC/OS universe
 ======================================
 
-This article assumes that you have set up an DC/OS and
+This article assumes that you have set up an DC/OS cluster and
```

Comment:
Since we're fixing this sentence up, it should be "a DC/OS" rather than "an DC/OS"